pkg/apis/eksctl.io/v1alpha4/types.go

```diff
@@ -137,6 +137,15 @@ func NewBoolFalse() *bool {
 	return &v
 }
 
+// BoolIsSetToTrue will only return true if v is not nil and true
+func BoolIsSetToTrue(v *bool) bool { return v != nil && *v }
```

Comment:
Maybe it could be named `FlagIsEnabled(...)`?

Reply:
Yeah, maybe `IsEnabled` would be a nice and short. Word 'flag' belongs to parts where it's about CLI flags (like `pkg/ctl/cmduits`), so let's not mix up words here it's the API package :)